proxy/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM haproxytech/haproxy-ubuntu:latest
```

Comment:
this may need to change once the project is deployed on our dev server. Try "FROM haproxy:latest". You can find specific versions on here [HAProxy](https://hub.docker.com/_/haproxy). I personally used haproxy:2.6 on my project.